src/components/SearchPanel/SearchPanel.module.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../global_styles/variables';
+
+.form {
+  width: 16.8rem;
+  display: flex;
+  justify-content: center;
+  gap: 0.35rem;
+
+  @include media-tablet {
+    width: 100%;
+  }
+
+  .submit {
```

Comment:
Please replace this class with `&__submit`. Currently you’ve increased the specificity of this class by putting it inside of the `.form` class for no reason